ship: refactor teams to match reorg (#7277)
sourcegraph/handbook: content/departments/cloud/index.md

> # Cloud Opreations Team

Reviewer: ```suggestion
# Cloud Operations Team
```

> # Cloud Opreations Team

Reviewer 2: ```suggestion
# Cloud Operations Team
```

> NOTE: **Cloud means single-tenant dedicated instances managed by Sourcegraph** _(for example `mycompany.sourcegraph.com`)_. Sourcegraph Cloud should not be confused with Sourcegraph.com which holds public and open source code. The Cloud and managed instance should be considered synonyms within these handbook pages.

> The Cloud team is the special focus team reporting directly to CEO modeled on _“if AWS were to offer ‘Managed Sourcegraph’ like they do Elasticsearch, Redis, PostgreSQL, etc., how would they do it?”_ The team is responsible for maintaining existing [managed instances](https://docs.sourcegraph.com/admin/install/managed) and building the next generation of them. The Cloud team has no other responsibilities.

Reviewer 3: some feedback to the Cloud section:

I think in the long run, we want Cloud to become the platform team and offload On-Call responsibility back to the home team. Is this something we want to capture here?

Author: We can. This is the team area, so as long as the team and leadership are aligned, it sounds like a good idea.

## How to contact the team and ask for help

- For emergencies and incidents, alert the team using Slack command `/genie alert [message] for cloud` and optionally tag the `@cloud-support` handle.
- For internal Sourcegraph teammates, join us in #ask-cloud-ops slack channel to ask questions or request help from our team.
- For [managed instance requests](#managed-instance-requests) or requests for help that requires action for the Cloud team engineers _(exp. coding, infrastructure change etc.)_ please create a GH issue and assign a `team/cloud` label. You can also post a follow up message on the #cloud slack channel
- You may tag the `@cloud-support` handle if you are looking for immediate attention, and it will notify our [on-call engineers](#on-call). Please avoid tagging/DM a specific teammate or the `@cloud-team` handle.

> ## Cloud Instances

Reviewer 4: This is an intentional terminology change? There are countless places we use "managed instances" and in reality most of the time people tend to type its abbreviation "MI", which is well-perceived. Where "CI" means... totally something different. @nelsonjr @rafax

Reviewer: My impression was that we use "Cloud Instance" more often than "Managed Instance" now (vs 1-year ago when this was written), but I don't feel strongly about this change - if you think we should stay with MI, happy to keep that.

Reviewer 4: A quick unscientific stats across our repos 😂 

<img width="596" alt="CleanShot 2023-07-05 at 09 30 25@2x" src="https://github.com/sourcegraph/handbook/assets/2946214/36de3706-d087-46e4-b026-347f1dd556ad">

<img width="609" alt="CleanShot 2023-07-05 at 09 31 01@2x" src="https://github.com/sourcegraph/handbook/assets/2946214/372f2648-0ef7-4df7-927d-c8d64e5eb4bb">

But I'm OK with any term, original intention was to make sure this ("Cloud Instances") is term  we are settling with, so whenever I see the another term, I know I should change/update them for consistency.

## How to contact the team and ask for help

- For emergencies and incidents, alert the team using Slack command `/genie alert [message] for cloud` and optionally tag the `@cloud-support` handle.

> - For internal Sourcegraph teammates, join us in #ask-cloud-ops slack channel to ask questions or request help from our team.

Reviewer 4: ```suggestion
- For internal Sourcegraph teammates, join us in #discuss-cloud-ops slack channel to ask questions or request help from our team.
```

- For [managed instance requests](#managed-instance-requests) or requests for help that requires action for the Cloud team engineers _(exp. coding, infrastructure change etc.)_ please create a GH issue and assign a `team/cloud` label. You can also post a follow up message on the #cloud slack channel
- You may tag the `@cloud-support` handle if you are looking for immediate attention, and it will notify our [on-call engineers](#on-call). Please avoid tagging/DM a specific teammate or the `@cloud-team` handle.

## Cloud Instances

### When to offer a Cloud Instance

> NOTE: Please first read [the customer-facing managed instance documentation](https://docs.sourcegraph.com/admin/install/managed) to understand what managed instances are and what we provide.

See below for the SLAs and Technical implementation details (including Security) related to managed instances.

> Please message [`#ask-cloud`](https://sourcegraph.slack.com/archives/C03JR7S7KRP) for any answers or information missing from this page.

Reviewer 4: ```suggestion
Please message [`#discuss-cloud-ops`](https://sourcegraph.slack.com/archives/C03JR7S7KRP) for any answers or information missing from this page.
```

3. If approved, then CE proceeds based on whether this is a standard or non-standard managed instance scenario:
   - For standard managed instance requests (i.e., new instance, no scale concerns, no additional security requirements), CE submits a request to the Cloud team using the corresponding [issue template](#managed-instance-requests) in the [sourcegraph/customer](https://github.com/sourcegraph/customer) repo.
   - For non-standard managed instance requests (i.e., any migrations, special scale or security requirements, or anything considered unusual), CE submits the opportunity to Tech Review before making a request to the Cloud team.

> 4. Message the team in [`#ask-cloud`](https://sourcegraph.slack.com/archives/C03JR7S7KRP).

Reviewer 4: ```suggestion
4. Message the team in [`#discuss-cloud-ops`](https://sourcegraph.slack.com/archives/C03JR7S7KRP).
```

Access to data from Cloud instances is governed by [Cloud Access Control Policy](https://docs.google.com/document/d/16tRqZDKhli4hZtfAkJG46Cj6dtqAw2s6QGw5fGM43BQ/edit?usp=sharing).

Long-term, we will collaborate with [Developer Infrastructure team](../engineering/teams/devinfra/index.md) (as owners of Sourcegraph observability) to support monitoring / observability solutions that are qualified for use with customer data.

### FAQ: What are Cloud plans for continuous deployment - how often do we deploy code to Cloud instances?

Cloud instances provisioned for customers run [released](../engineering/dev/process/releases/index.md#releases) Sourcegraph versions and are currently updated at least once a month (for minor releases), unless we need to deploy a patch release.

> Sourcegraph-owned instances are continuously deployed (with versions that weren't officially released), [Developer Infrastructure team](../engineering/teams/devinfra/index.md) owns continuous deployment to those environments.

Reviewer 4: ```suggestion
Sourcegraph-owned instances can opt-in to receive continuous upgrades with versions that weren't officially released, e.g., the latest commit on `main` branch, release branches. We own continuous deployment to those environments.
```

Access to data from Cloud instances is governed by [Cloud Access Control Policy](https://docs.google.com/document/d/16tRqZDKhli4hZtfAkJG46Cj6dtqAw2s6QGw5fGM43BQ/edit?usp=sharing).

Long-term, we will collaborate with [Developer Infrastructure team](../engineering/teams/devinfra/index.md) (as owners of Sourcegraph observability) to support monitoring / observability solutions that are qualified for use with customer data.

### FAQ: What are Cloud plans for continuous deployment - how often do we deploy code to Cloud instances?

Cloud instances provisioned for customers run [released](../engineering/dev/process/releases/index.md#releases) Sourcegraph versions and are currently updated at least once a month (for minor releases), unless we need to deploy a patch release.

> Sourcegraph-owned instances are continuously deployed (with versions that weren't officially released), [Developer Infrastructure team](../engineering/teams/devinfra/index.md) owns continuous deployment to those environments.

Reviewer 3: ```suggestion
Sourcegraph-owned instances can opt-in to receive continuous upgrades with versions that weren't officially released, e.g., the latest commit on `main` branch, release branches. We own continuous deployment to those environments.
```
